zk/assets/configuration/spec.yaml

```diff
@@ -34,9 +34,13 @@ files:
               If `expected_mode` is defined, a service check is sent where the
               status is determined by whether the current mode matches the expected.
 
-              Available options are: `leader`, `follower`, `standalone`.
+              Available options are: `leader`, `follower`, `standalone` or an array of one or more of these.
```

Comment:
```suggestion
              Available options are: `leader`, `follower`, `standalone`, or an array of one or more of the following:
```

Reply:
`these` actually refers to the preceding `leader, follower, standalone`, so I don't think this change works